src/System.ComponentModel.TypeConverter/src/System/ComponentModel/InterlockedBitVector32.cs

```diff
@@ -50,8 +51,10 @@ public static int CreateMask(int previous)
             return previous == 0 ? 1 : previous << 1;
         }
 
+        [ExcludeFromCodeCoverage]
         public override bool Equals(object o) => o is InterlockedBitVector32 ? _data == ((InterlockedBitVector32)o)._data : false;
 
+        [ExcludeFromCodeCoverage]
```

Comment:
Why did you add these attributes?

Reply:
This was dead code and I wasn't sure if I should get rid of it though. The constructor was unused and can definitely be deleted.

I audited the code and the only use of `InterlockedBitVector32` is in https://github.com/dotnet/corefx/blob/master/src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs#L64

No uses of state call `Equals` or `GetHashCode`, so this is dead:
https://github.com/dotnet/corefx/blob/master/src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs#L64

Reply:
We shouldn't be adding such attributes, especially if it's dead code (how would someone know to look at these in the future for possible removal?).

I wouldn't remove the code, though.  structs like this are supposed to override GetHashCode and Equals.  If someone ended up using these in the future and we removed them, the caller would get the base implementation's that is really expensive.  I'd prefer to err on the side of just leaving them as-is.

Reply:
Okay, `GetHashCode` and `Equals` have returned! I do think that the ExcludeFromCodeCoverageAttribute is helpful here, so I kept it and added comments

Reply:
> I do think that the ExcludeFromCodeCoverageAttribute is helpful here,

Why? It's effectively a lie. We should only be using it for things that we don't care about (e.g. test assemblies) rather than trying to improve a vanity metric.